src/main/java/com/twilio/http/Request.java

```diff
@@ -216,16 +216,17 @@ public void addQueryDateRange(final String name, final Range<LocalDate> range) {
      * Add query parameters for date ranges.
      *
      * @param name  name of query parameter
-     * @param range date range
+     * @param lowerBound date lowerBound
+     * @param upperBound date upperBound
      */
-    public void addQueryDateTimeRange(final String name, final Range<ZonedDateTime> range) {
-        if (range.hasLowerBound()) {
-            String value = range.lowerEndpoint().withZoneSameInstant(ZoneOffset.UTC).format(DateTimeFormatter.ofPattern(QUERY_STRING_DATE_TIME_FORMAT));
+    public void addQueryDateTimeRange(final String name, final ZonedDateTime lowerBound, final ZonedDateTime upperBound) {
```

Comment:
This is being called as `request.addQueryDateTimeRange("StartTime", startTimeBefore, startTimeAfter);`, but `startTimeBefore` is really an _upper_ bound.

Given that this was able to pass without any tests breaking, tests need to be updated to catch this.

Reply:
Nice catch! Will fix.